server/plugin_release.go

```diff
@@ -127,16 +127,20 @@ func getReleaseByTag(ctx context.Context, client *GithubClient, owner, repositor
 // createTag creates a new tag at the given commit for the repository.
 // Returns ErrTagExists if tag already exists, nil if successful and an error otherwise.
 func createTag(ctx context.Context, client *GithubClient, owner, repository, tag, commitSHA string) error {
-	refs, _, err := client.Git.GetRefs(ctx, owner, repository, fmt.Sprintf("tags/%s", tag))
+	tagRef := fmt.Sprintf("tags/%s", tag)
+	refs, _, err := client.Git.GetRefs(ctx, owner, repository, tagRef)
```

Comment:
go-github will throw an error and a response when there is no ref found. I ran into this inconsistency with the rest of the library already a few times.

Reply:
@metanerd yep that's correct, which is why i also check for the `err` here and check for the `StatusNotFound` status code, as I want to create the tag if it doesn't exist.

Also even if `refs` is `nil` it's still iterable and the loop below should still work fine.

Was that the concern here?

Reply:
Oh my, yes! Nicely done!
![guy](https://user-images.githubusercontent.com/9913860/83520187-90114500-a4dd-11ea-92ec-d61bb54194a3.jpg)